charts/datadog/values.yaml

```diff
@@ -632,7 +632,7 @@ datadog:
 
   containerImageCollection:
     # datadog.containerImageCollection.enabled -- Enable collection of container image metadata
-    enabled: false
+    enabled: true
```

Comment:
I didn't think of this earlier; do you think it's worth adding a minimum agent version in the description?

Reply:
This sounds like something we should have done for all the other options.
For this parameter to work, we need DataDog/datadog-agent#16847. Hence `7.46+`.
Container images collection is a feature which is much older than that. But the way to enable it was different.
So, 7.46+ is the minimum version so that it is enabled by this version of the chart.

Here is the change in the description: https://github.com/DataDog/helm-charts/pull/1228/commits/b0b335b6979cb438d01e9c34911990bba5ad53bd

Reply:
thanks